packages/cashier/src/modules/cashier-onboarding/cashier-onboarding.scss

```diff
@@ -0,0 +1,9 @@
+.cashier-onboarding-page {
+    height: calc(100vh - 17rem);
+    padding-left: 2.4rem;
+    padding-right: 2.4rem;
+
+    @include mobile {
+        height: calc(100vh - 8rem);
```

Comment:
Unfortunately, we have to keep the `height` hardcoded and can't rely on flex and responsive until we refactor the layouting.